# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xpdf/tetex3.patch.t1lib
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- T1Lib-1.3/doc/t1lib_doc.tex.vanilla	2005-02-27 00:08:48.000000000 +0100
+++ T1Lib-1.3/doc/t1lib_doc.tex	2005-02-27 00:09:07.000000000 +0100
@@ -33,7 +33,7 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% Added by NGUYEN-DAI Quy and edited by RMz)
 %%
-\ifx\pdfoutput\undefined
+\ifnum\pdfoutput=0
   \usepackage[dvips]{graphicx}
 %  \usepackage[dvips,a4paper,breaklinks]{hyperref}
 \else
